### PR TITLE
Add OpenVEX JSON-LD Context

### PR DESCRIPTION
**Pull request by puerco**
This PR adds the JSON-LD context file to the spec repository

~Requires approval of [OPEV-0014](https://github.com/openvex/community/issues/14) :heavy_check_mark:  and [OPEV-0015](https://github.com/openvex/community/issues/15) :heavy_check_mark:~

Signed-off-by: Adolfo García Veytia (Puerco) <puerco@chainguard.dev>
Signed-off-by: Adolfo García Veytia (Puerco) <puerco@stacklok.com>

### CODE REVIEW

File: ns/v1/README.md
Comment (SecurityCRob) [inline ns/v1/README.md:1]: Is there any thought to keeping version number consistent across spec, tools, etc?  This is the v1 for the json-ld context, but would we want to rev it to match the most current versions across the other efforts?

**Comment by sudo-bmitch** [comment]
> /hold **DO NOT MERGE**

Is this still on hold?

**Comment by puerco** [comment]
Now that we have the spec merged, I'll return to this

**Comment by cpanato** [comment]
@puerco ping

**Comment by puerco** [comment]
Ugh I forgot this PR was open :cry: 

I pushed another commit with the rest of the context file I've with all improvements I been working on for the past _year_ (yikes). It is closer to being complete but we need some coverage tests. But for now, this version will allow us to build some infra to serve the context file.

**Comment by puerco** [comment]
Here's the current N-Quads from one the samples in the vexctl repo:

```xml
<https://openvex.dev/docs/public/vex-d7653e1c3ccc092c02e2cfd2b12549be29264ff28f9f704f599613b007584dbe> <https://openvex.dev/ns/v0.2.0#author> "Wolfi J Inkinson" .
<https://openvex.dev/docs/public/vex-d7653e1c3ccc092c02e2cfd2b12549be29264ff28f9f704f599613b007584dbe> <https://openvex.dev/ns/v0.2.0#role> "Document Creator" .
<https://openvex.dev/docs/public/vex-d7653e1c3ccc092c02e2cfd2b12549be29264ff28f9f704f599613b007584dbe> <https://openvex.dev/ns/v0.2.0#statement> _:b0 .
<https://openvex.dev/docs/public/vex-d7653e1c3ccc092c02e2cfd2b12549be29264ff28f9f704f599613b007584dbe> <https://openvex.dev/ns/v0.2.0#timestamp> "2023-12-04T21:55:19.945014473-06:00"^^<http://www.w3.org/2001/XMLSchema#dateTime> .
<https://openvex.dev/docs/public/vex-d7653e1c3ccc092c02e2cfd2b12549be29264ff28f9f704f599613b007584dbe> <https://openvex.dev/ns/v0.2.0#version> "1"^^<http://www.w3.org/2001/XMLSchema#integer> .
<pkg:apk/distro/git@2.39.0-r1?arch=x86_64> <https://openvex.dev/ns/v0.2.0#subcomponent> _:b1 .
_:b0 <https://openvex.dev/ns/v0.2.0#product> <pkg:apk/distro/git@2.39.0-r1?arch=armv7> .
_:b0 <https://openvex.dev/ns/v0.2.0#product> <pkg:apk/distro/git@2.39.0-r1?arch=x86_64> .
_:b0 <https://openvex.dev/ns/v0.2.0#status> "fixed"^^<https://openvex.dev/ns/v0.2.0#vexStatus> .
_:b0 <https://openvex.dev/ns/v0.2.0#timestamp> "2023-12-04T21:55:19.945015302-06:00"^^<http://www.w3.org/2001/XMLSchema#dateTime> .
_:b0 <https://openvex.dev/ns/v0.2.0#vulnerability> _:b2 .
_:b1 <https://openvex.dev/ns/v0.2.0#hash> "3e6808a3a2c5b33fbbd6c3893731a0c3480a591ceae11f40f26baecd592061a9" .
_:b2 <https://openvex.dev/ns/v0.2.0#name> "CVE-2014-123456"^^<https://openvex.dev/ns/v0.2.0#vulnName> .

```

And here is the same document visualized, I think it's looking good:

image

here's a link to the jsonld playground in case folks want to play with it:  https://tinyurl.com/2cblblwo

**Comment by puerco** [comment]
@sudo-bmitch @SecurityCRob @sudo-bmitch PTAL

File: ns/v1/README.md
Comment (puerco) [reply ns/v1/README.md:1]: I pushed a new commit moving the context to the proper directory (`v0.2.0`) , also updated the README and top level link

**Review by cpanato — APPROVED** [review]

**Comment by sudo-bmitch** [comment]
> @sudo-bmitch PTAL

No objection from me, but I'm far from an expert on this.

**Comment by puerco** [comment]
OK, let's merge and iterate, thanks!

**Pull request merged**